movieland-persistence-jdbc/src/main/java/com/gromoks/movieland/dao/config/SpringConfig.java

```diff
@@ -0,0 +1,9 @@
+package com.gromoks.movieland.dao.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.scheduling.annotation.EnableScheduling;
+
+@Configuration
+@EnableScheduling
+public class SpringConfig {
```

Comment:
Can be populated in .xml, you have root context configured there

Reply:
remove xml from module, use only configuration here

Reply:
xml was removed